Tests/LibWeb/Layout/expected/blockify-layout-internal-box-without-crashing.txt

```diff
@@ -13,7 +13,7 @@ ViewportPaintable (Viewport<#document>) [0,0 800x600]
     PaintableWithLines (BlockContainer<BODY>) [8,8 784x4]
       PaintableWithLines (TableWrapper(anonymous)) [8,8 4x4]
         PaintableBox (Box<TABLE>) [8,8 4x4]
-          PaintableBox (Box<TBODY>) [8,8 0x0] overflow: [8,8 2x2]
+          PaintableBox (Box<TBODY>) [10,10 0x0] overflow: [8,8 2x2]
```

Comment:
Hmm, this still has overflow somehow.

Reply:
This one has `position: absolute` set on the `td`, which is causing overflow up the chain